docs/WindowsBuild.md

```diff
@@ -152,8 +160,13 @@ cmake -G "Ninja" "%swift_source_dir%/swift"^
  -DSWIFT_INCLUDE_TESTS=FALSE^
  -DCMAKE_C_COMPILER="%llvm_bin_dir%/clang-cl.exe"^
  -DCMAKE_CXX_COMPILER="%llvm_bin_dir%/clang-cl.exe"^
- -DCMAKE_C_FLAGS="-fms-compatibility-version=19.00 /Z7"^
- -DCMAKE_CXX_FLAGS="-fms-compatibility-version=19.00 -Z7" ^
+ -DCMAKE_C_FLAGS="-fms-compatibility-version=19.00 /Z7 -Wno-c11-extensions -Wno-language-extension-token -Wno-reserved-id-macro -Wno-disabled-macro-expansion"^
+ -DCMAKE_CXX_FLAGS="-fms-compatibility-version=19.00 -Z7 -Wno-c++98-compat -Wno-c++98-compat-pedantic -Wno-c11-extensions -Wno-old-style-cast -Wno-language-extension-token -Wno-reserved-id-macro -Wno-disabled-macro-expansion"^
+ -DCMAKE_EXE_LINKER_FLAGS:STRING="/machine:x64 /INCREMENTAL:NO"^
+ -DCMAKE_MODULE_LINKER_FLAGS="/machine:x64 /INCREMENTAL:NO"^
+ -DCMAKE_SHARED_LINKER_FLAGS="/machine:x64 /INCREMENTAL:NO"^
+ -DCMAKE_STATIC_LINKER_FLAGS="/machine:x64 /INCREMENTAL:NO"^
```

Comment:
The `/machine` flag shouldn't be needed.  That is addressed with using the right Developer Command Prompt environment.

Reply:
I can removed this if you want me to. I simply preserved what cmake generated by default.

Reply:
Yeah, lets simplify it.